src/notifier.rs

```diff
@@ -94,33 +94,36 @@ fn send_message(task: &DeployTask, config: &RepoConfig, status: TaskState) {
     };
 
     let client = Client::new();
-    println!("[{}]: notifier: sending {} message to {}",
-             &task.id,
-             &status,
-             &notify_url);
 
     // Spawn a new thread to send the message so we don't block the task
     let task_id = task.id.clone();
-    let notify_url = notify_url.clone();
+    let notifiers = notify_url.clone();
     let secret = task.secret.clone();
```

Comment:
`notify_url` should probably be renamed to `notifiers` or `notify_urls` in all the places since it's no longer a singular thing.